src/controller/commentController.js

```diff
@@ -0,0 +1,92 @@
+import Comments from "../model/commentModel";
+import Posts from "../model/postModel";
+import Users from "../model/userModel";
+
+export const addComment = async (req, res) => {
+  try {
+    const {Post_Id} = req.params;
```

Comment:
Please use camelCase like `postId` for this case

Reply:
Resolved camelCase naming